src/drt/doc/Distributed.md

```diff
@@ -56,6 +56,15 @@ A shared folder on a NFS is used to share routing updates between the leader and
 
 [https://www.digitalocean.com/community/tutorials/how-to-set-up-an-nfs-mount-on-ubuntu-20-04](https://www.digitalocean.com/community/tutorials/how-to-set-up-an-nfs-mount-on-ubuntu-20-04)
 
+The main steps on the NFS server machine are:
+
+1. run ` sudo apt install nfs-kernel-server ` and ` sudo apt install nfs-common `
+1. In the etc/exports file, add the following line replacing \${PATH} with the actual shared directory path you choose: ` ${PATH} 10.128.0.0/255.255.0.0(rw,no_subtree_check,no_root_squash) `
```

Comment:
Is `10.128.0.0` generic for all users?

Reply:
not really. it depends on the private network local ip range.

Reply:
Then the docs should explain that the users needs to update this as well as PATH.